lib/miq_expression/field.rb

```diff
@@ -17,6 +17,10 @@ def self.parse!(field)
     parse(field) or raise ParseError, field
   end
 
+  def self.filter_has_many(cols)
+    cols ? cols.select { |_, column| parse(column).try(:plural?) } : []
```

Comment:
Very nice `+4/-26`

I like having this method here but the format of `cols` is very ui/reports specific. (And only used in 1 controller method)

@imtayadeway you want this documented, or you want to move this closer to the ui?